packages/apps/job-launcher/server/src/modules/cron-job/cron-job.service.ts

```diff
@@ -88,6 +92,7 @@ export class CronJobService {
         try {
           await this.jobService.createEscrow(jobEntity);
         } catch (err) {
+          jobEntity.failedReason = ErrorEscrow.NotCreated;
```

Comment:
Pass the failed reason into handleProcessJobFailure.
Also we should generate an errorId (GUID) and include it in the log and in the failed reason, so that later when we try to find out what happened we can identify the log

Reply:
Error setting up escrow. Error ID: ab83a915-5856-49cd-8e0b-386f7998b4b5, Job ID: 44, Reason: Escrow has not been setup (Error ID: ab83a915-5856-49cd-8e0b-386f7998b4b5), Message: Manifest validation failed